### PR TITLE
doc: add multi tenancy

### PR DESCRIPTION
**Pull request by alichaddad**
Closes

**What changed?**
Added multi tenancy doc

### CODE REVIEW

File: website/docs/enterprise/multi-tenancy.mdx [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+WGE multi tenancy expands on the multi tenancy feature provided by `flux`. In addition to creating the necessary Kubernetes tenancy resources that `flux` adds, multi tenancy in WGE also adds the following:
+- Defining tenancy using a single yaml file that serves as a source of truth for the organization
+- Makes use of WGE policy features to enforce non Kubernetes native permissions
+
+
+## How it works
+
+`gitops` command line tool is responsible for creating the multi tenancy resources. The tool is distributed as part of WGE offering. It reads the definitions of a yaml file and can either apply the necessary changes directly to the cluster or output it to stdout so it can be saved into a file and pushed to a repo to be reconcilled by `flux`.
+
+To make use of the policy features, [policy agent](../policy/intro.mdx) needs to be installed in the necessary cluster(s).
```
Comment (foot) [inline website/docs/enterprise/multi-tenancy.mdx:22]: 👍

File: website/docs/enterprise/multi-tenancy.mdx [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+To check the resources that would be deployed first use the `export` flag:
+
+```bash
+gitops create tenants --from-file tenants.yaml --export
+```
+
+Applying the resources through the command line is not usually recommended. For WGE the recommended way is to commit the result of the `create tenants` command to source control and let `flux` handle deployment. To achieve that you can save the result of the `export` to a file:
+
+```bash
+gitops create tenants --from-file tenants.yaml --export > tenants.yaml
```
Comment (foot) [inline website/docs/enterprise/multi-tenancy.mdx:96]:
```suggestion
gitops create tenants --from-file tenants.yaml --export > clusters/management/tenants.yaml 
```

**Review by foot — APPROVED** [review]
This all looks good to me :), nice one

File: website/docs/enterprise/multi-tenancy.mdx [hunk 3]
```diff
@@ -0,0 +1,102 @@
...
+
+# Multi tenancy <TierLabel tiers="enterprise" />
+
+Multi tenancy provides users with the ability to define boundaries to multiple engineering teams working on a single cluster. Through a simple interface it adds permissions to the necessary Kubernetes resources to make it easy for customers to manage their multiple tenants.
+
+WGE multi tenancy expands on the multi tenancy feature provided by `flux`. In addition to creating the necessary Kubernetes tenancy resources that `flux` adds, multi tenancy in WGE also adds the following:
+- Defining tenancy using a single yaml file that serves as a source of truth for the organization
+- Makes use of WGE policy features to enforce non Kubernetes native permissions
+
+A ‘Tenant’ gives you the control to define:
```
Comment (davidstauffer) [inline website/docs/enterprise/multi-tenancy.mdx:17]: Added some Marketing blurb :)

**Pull request merged**